tools/mOTUs/motus_profiler.xml

```diff
@@ -0,0 +1,221 @@
+            $additional_options.advanced.result_as_counts
+            $additional_options.advanced.ncbi_taxonomy_ids
+            $additional_options.advanced.biom_format
+            $additional_options.advanced.cami_format_option
+            $additional_options.advanced.full_rank_taxonomy
+            $additional_options.advanced.taxonomic_level_metaphlanType
+            $additional_options.advanced.taxonomic_level_option
+        #end if
+        > '${mOTUs_profile}'
+    ]]></command>
+    <inputs>
+        <section name="inputs" title="Inputs" expanded="true">
+            <conditional name="input_type">
+                <param name="selector" type="select" label="Input file(s)">
+                    <option value="reads" selected="True">fastq,fastq.gz file(s)</option>
+                    <option value="SAM_BAM_input">SAM/BAM input file(s) generated by motus map_tax</option>
+                    <option value="mgc_read_count">mgc_read_count file(s) generated by motus cal-mgc</option>
+                </param>
+                <when value="reads">
+                    <conditional name="reads_type">
+                        <param name="selector" type="select" format="fastq,fastq.gz" label="Choose input file(s)">
+                            <option value="paired" selected="true">fastq,fastq.gz Forward and Reverse Reads</option>
+                            <option value="single" selected="true">fastq,fastq.gz Unpaired reads</option>
+                        </param>
+                        <when value="paired">
+                            <param name="forward_reads" type="data" format="fastq,fastq.gz" label="Forward Reads" help="-f Input file(s) for reads in forward orientation, fastq(.gz)-formatted"/>
+                            <param name="reverse_reads" type="data" format="fastq,fastq.gz" label="Reverse Reads" help="-r Input file(s) for reads in reverse orientation, fastq(.gz)-formatted"/>
+                        </when>
+                        <when value="single">
+                            <param name="unpaired_reads" type="data" format="fastq,fastq.gz" label="Unpaired reads" help="-s input file(s) for unpaired reads, fastq(.gz)-formatted"/>
+                        </when>
+                    </conditional>
+                </when>
+                <when value="SAM_BAM_input">
+                    <param name="SAM_BAM_input" type="data" format="sam,bam" label="motus map_tax SAM or BAM file" help="-i SAM or BAM input file(s) generated by motus map_tax"/>
+                </when>
+                <when value="mgc_read_count">
+                    <param name="mgc_count" type="data" format="mgc" label="mgc read count file)(s)" help="-m mgc reads count file generated by motus calc_mgc"/>
+                </when>
+            </conditional>
+                <param name="sample_name" type="text" label="Sample name" help="sample name can contain letters and/or numbers" default_value="unnamed sample"/>
```

Comment:
something wrong with indentation.

The VSC plugin has an autoformat option for you. CRTL+SHIFT+P I think ...